### DISCUSSION TITLE
Appwrite CLI working with Multiple Projects

### DISCUSSION THREAD

**Discussion by lohanidamodar** [appwrite/appwrite#991] [Ideas]
Working with multiple Appwrite projects at once is common. Right now each time we want CLI to work with another project, we need to run Appwrite init again every time, this is painful if we want to keep switching between projects.

Suggested Solutions
## Using `.appwrite.yml` file
Create `.appwrite.yml` file in a local directory with the required settings (project id, api key, host) and execute Appwrite CLI from that folder, it will look for `.appwrite.yml` file and load the settings from there. If the file is not present it will use the default settings from preferences set using `appwrite init`

## Using alias
We could save multiple project settings by providing Alias in preferences

1. Appwrite init —alias=project1 ...... (like this we can setup multiple alias)
2. appwrite select project1 (will select an alias project to work with)
3. All further appwrite commands will take project1 settings

**Comment by lohanidamodar** [comment]
@christyjacob4 verify that I got idea of using `.appwryte.yml` file right.

**Comment by kodumbeats** [comment]
While copypasting `projectIDs` is cumbersome, what's more cumbersome is creating an API key with appropriate scopes for each project. Is there any creative way we can solve this problem too?

**Reply by eldadfux** [reply]
For me probably the solution is account level API Key with access to all projects and user action from the Appwrite console. The biggest issue here as I see it, is how we're gonna make it accessible to devs, and how do we name in an intuitive way to not conflict with current auth solutions.

If this will be available - switching between project will not be a technical challenge.
